README.md

```diff
@@ -22,3 +22,25 @@ Tested on Ubuntu 18.04 and Debian 10.
 * Put all these files in `/var/www/html`
 * Run `sudo setup.sh`
 * Visit http://localhost in your browser
+
+FAQ
+----
+**Can you delete it when you are done with it?**
+
+Yes. For any wiki you create, you will see a `Delete` link in the `Action` column of this table: https://patchdemo.wmflabs.org/
+
+**How long do the Patch demo wiki instances last for?**
+
+There is no definitive time after which wikis will automatically be deleted. With this said, we make no guarantees about how long they will continue to exist. A Patch demo wiki you've created could be deleted if we need to free up disk space to create space for new ones.
+
+**Can Patch demo wikis be named?**
+
+Wikis can not been named *within* Patch demo. Patch demo wikis currently inherit the name of the patch on which they are built.
+
+**Is it possible to add extensions that are in development?**
+
```

Comment:
These will be considered on a case-by-case basis, but will generally be allowed as long as they don't interfere with other teams' ability to test in a production-like environment.

Reply:
Done.